src/v2i-hub/MapPlugin/src/MapPlugin.cpp

```diff
@@ -8,6 +8,7 @@ namespace MapPlugin {
 		AddMessageFilter(IVPMSG_TYPE_SIGCONT, "ACT", IvpMsgFlags_None);
 		SubscribeToMessages();
 		errThrottle.set_Frequency(std::chrono::minutes(30));
+		
```

Comment:
Is Subscribe to messages on Line 9 here calling the same subscription to TimeSync messages? This should also have a check for simulation mode

Reply:
It will subscribe to all message filters added. This means that for plugins that are already adding message filters we can call subscribe at it will subscribe to both time sync and all other appropriate message filters. For plugins that do not add any message filters we have to check for simulation mode since attempting to call subscribe without any message filters will return an exception